shared/router-v2/router.desktop.js

```diff
@@ -50,7 +50,7 @@ class AppView extends React.PureComponent<any> {
           fullHeight={true}
           style={selectedTab ? styles.contentArea : styles.contentAreaLogin}
         >
-          <Header options={descriptor.options} onPop={childNav.pop} allowBack={index !== 0} />
+          <Header options={descriptor.options} onPop={() => childNav.pop()} allowBack={index !== 0} />
```

Comment:
what was the problem here?

Reply:
onPop passes an event, childNav.pop plumbs through params so its possible you can do childNav.pop(3) which goes back 3 levels or whatever